### PR TITLE
Hirer onboarding

### PR DESCRIPTION
**Pull request by 0xRomain**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
     "yup": "^0.32.11"
   },
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -p 3000",
```
Comment (0xRomain) [inline package.json]: not necessaray anymore can be dynamic if your .env is setup with the same NEXT_PUBLIC_ROOT_DOMAIN

File: src/modules/BuilderPlace/models/BuilderPlace.ts [hunk 2]
```diff
@@ -36,6 +37,15 @@ var builderPlace = new Schema({
     type: String,
     default: 'pending',
   },
+  preferredWorkType: {
```
Comment (0xRomain) [inline src/modules/BuilderPlace/models/BuilderPlace.ts]: preferredWorkTypeS as there can be several values

File: src/modules/BuilderPlace/components/form/MintTlidFormStep2.tsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+import SubmitButton from '../../../../components/Form/SubmitButton';
+import HelpPopover from '../../../../components/HelpPopover';
+import { useContext } from 'react';
+import TalentLayerContext from '../../../../context/talentLayer';
+
+interface IFormValues {
+  handle: string;
+}
+
+function MintTlidFormStep2({
```
Comment (0xRomain) [inline src/modules/BuilderPlace/components/form/MintTlidFormStep2.tsx]: could be nice to find a way to not duplicate the tlId form

File: src/modules/BuilderPlace/actions.ts [hunk 4]
```diff
@@ -59,16 +75,16 @@ export const createBuilderPlace = async (data: CreateBuilderPlaceAction) => {
     const newBuilderPlace = new BuilderPlace({
       _id: new mongoose.Types.ObjectId(),
       name: data.name,
+      presentation: data.presentation,
       subdomain: data.subdomain,
+      preferredWorkType: data.preferredWorkType,
       customDomain: 'null',
       logo: 'a',
```
Comment (0xRomain) [inline src/modules/BuilderPlace/actions.ts]: could be nice to clean this code

File: src/pages/landing/onboarding/index.tsx [hunk 5]
```diff
@@ -1,43 +1,198 @@
...
+  const router = useRouter();
+
+  const initialValues: IFormValues = {
+    name: '',
+    presentation: '',
+    preferred_work_type: [PreferredWorkType.jobs],
+    image_url: '',
+  };
+
+  const validationSchema = Yup.object({
```
Comment (0xRomain) [inline src/pages/landing/onboarding/index.tsx:27]: did you have a problem here ?

Comment (quent043) [reply src/pages/landing/onboarding/index.tsx:27]: Just did this for testing upload, so I wouldn't have to refill all fields

**Pull request merged**